src/index.ts

```diff
@@ -79,6 +76,9 @@ class DXScannerCommand {
       process.exit(1);
     });
 
+    // notify about new version of CLI
+    cmder.on('close', () => this.notifyUpdate);
```

Comment:
notify on program event "close" (id doesn't work for now because it's not possible to override "close" event of the parent process (https://github.com/tj/commander.js/blob/master/index.js#L796-L803)

Reply:
`cmder.on('command:*', () => this.notifyUpdate);` doesn't work too